api/scripts/send-invitations-to-sco-organizations.js

```diff
@@ -1,118 +1,75 @@
-/* eslint-disable no-console */
-// Usage: BASE_URL=... PIXMASTER_EMAIL=... PIXMASTER_PASSWORD=... node send-invitations-to-organizations.js path/file.csv
-// To use on file with columns |externalId, email|
+const bluebird = require('bluebird');
```

Comment:
Je suggère de nommer le script send-join-sco-organization-invitations-to-user.js